alf/algorithms/dynamic_action_repeat_agent.py

```diff
@@ -346,7 +345,10 @@ def preprocess_experience(self, rl_exp):
 
             clip = self._reward_normalizer._clip_value
             if clip > 0:
-                reward = torch.clamp(reward, -clip, clip)
+                # The clip value is for single-step rewards, so we need to multiply
+                # it with the repeated steps.
+                clip = clip * repeats
+                reward = torch.max(torch.min(clip, reward), -clip)
```

Comment:
why not keep using clamp()?

Reply:
> why not keep using clamp()?

clamp only supports scalar min and max as I know. After multiplying `repeats`, `clip` is a tensor.